### PR TITLE
JE Completed priceChange form, table, and tests

### LINKED ISSUE #14: FEATURE: Automate market price changes
### Automate Market Price Changes
Every day, the market price for cows should fluctuate within predefined bounds. Market price fluctuation should incentivize players to be greedy and purchase more cows when the price dips, perpetuating the tragedy of the commons.

**Requirements**

- [x] Admin should be able to define standard deviation for the price of cows to fluctuate.
- [x] Cow price should fluctuate daily, centered around an average that the admin will define.

**Stretch goals**
- [x]  Market price changes should realistically implement supply/demand

### PR DESCRIPTION
**Pull request by MrJacobEisner**
In this PR, we implemented changes to form and table to show the priceChange field. We also implemented the tests. This is for issue https://github.com/ucsb-cs156-s23/proj-happycows-s23-6pm-3/issues/14

BEFORE:

* [Commons Form Storybook](https://ucsb-cs156-s23.github.io/proj-happycows-s23-6pm-3/storybook/?path=/story/components-commons-commonsform--uncontrolled)
* [Commons Table Storybook](https://ucsb-cs156-s23.github.io/proj-happycows-s23-6pm-3/storybook/?path=/st

image
image
AFTER:

* [Commons Form Storybook](https://ucsb-cs156-s23.github.io/proj-happycows-s23-6pm-3/prs/35/storybook/?path=/story/components-commons-commonsform--uncontrolled)
* [Commons Table Storybook](https://ucsb-cs156-s23.github.io/proj-happycows-s23-6pm-3/prs/35/storybook/?path=/story/components-commons-commonstable--three-commons-admin)

image
image

### CODE REVIEW

**Review by Biggergig — DISMISSED** [review]

File: frontend/src/main/components/Commons/CommonsTable.js [hunk 1]
```diff
@@ -22,84 +24,83 @@ export default function CommonsTable({ commons, currentUser }) {
...
+            Header: "Cow Price",
+            accessor: (row) => row.commons.cowPrice,
+            id: "commons.cowPrice",
         },
         {
-            Header:'Cow Price',
-            accessor: row => row.commons.cowPrice,
-            id: 'commons.cowPrice'
+            Header: "Price Change",
+            accessor: "commons.priceChange",
```
Comment (awaang) [inline frontend/src/main/components/Commons/CommonsTable.js]: Why does Price Change not need an id? Is that on purpose? Also, why is the accessor different from the others?

**Review by awaang — CHANGES_REQUESTED** [review]

File: frontend/src/main/components/Commons/CommonsTable.js (hunk 1 above)
Comment (MrJacobEisner) [reply frontend/src/main/components/Commons/CommonsTable.js]: I tried playing around with this, I believe this is correct as can be seen on lines 39 and 43 in the same file.

**Review by awaang — DISMISSED** [review]

**Review by pconrad — CHANGES_REQUESTED** [review]
I also filled in the missing storybook entries for this PR and noticed, as I did, that the screenshot and the storybook reveal something missing: the table doesn't have any entry in it for price change.   That column is empty.

This could just be a storybook fixtures issue, or it could be something else.  Could you take a look?

File: frontend/src/main/components/Commons/CommonsTable.js [hunk 2]
```diff
@@ -22,84 +24,83 @@ export default function CommonsTable({ commons, currentUser }) {
...
+            Header: "Cow Price",
+            accessor: (row) => row.commons.cowPrice,
+            id: "commons.cowPrice",
         },
         {
-            Header:'Cow Price',
-            accessor: row => row.commons.cowPrice,
-            id: 'commons.cowPrice'
+            Header: "Price Change",
+            accessor: "priceChange",
```
Comment (MrJacobEisner) [inline frontend/src/main/components/Commons/CommonsTable.js:52]: Fixed the bug mentioned by Professor Conrad!

**Review by pconrad — APPROVED** [review]

File: frontend/src/main/components/Commons/CommonsTable.js [hunk 3]
```diff
@@ -22,84 +24,83 @@ export default function CommonsTable({ commons, currentUser }) {
     // Stryker enable all
```
Comment (pconrad) [inline frontend/src/main/components/Commons/CommonsTable.js]: Should be 
```suggestion
    // Stryker restore all
```

**Review by pconrad — CHANGES_REQUESTED** [review]
One small change (enable->restore) then LGTM

**Review by epuzio — APPROVED** [review]
Looks good

**Review by awaang — APPROVED** [review]

**Comment by awaang** [comment]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**